store/tikv/commit.go

```diff
@@ -91,6 +91,14 @@ func (actionCommit) handleSingleBatch(c *twoPhaseCommitter, bo *Backoffer, batch
 				zap.Uint64("txnStartTS", c.startTS),
 				zap.Stringer("info", logutil.Hex(rejected)))
 
+			// Do not retry for a txn which has a too large MinCommitTs
+			// 3600000 << 18 = 943718400000
```

Comment:
1000 hours? Is there any particular reason for choosing this value?

Reply:
I think it's 1 hour. TiDB will retry this in a high frequency and the stack will got overflowed in less than 1 minute. Maybe we can adjust it to 1 minute?

Reply:
Got it, I think 1 hour is OK. Maybe 1 minute is too small.